DEVELOPERS.md

```diff
@@ -172,6 +172,8 @@ You must be using NPM 7 or higher.
 
 ### Getting started
 
+From the supabase/apps directory: 
```

Comment:
Change this to supabase root directory

When running npm install/npm run dev from the supabase root, you can work on the apps/www folder and hot reload will show your changes.